plugins/jfrog-artifactory/README.md

```diff
@@ -2,15 +2,23 @@
 
 This plugin will show you information about your container images within Jfrog Artifactory registry
 
-## Getting started
+## Table of Contents
+
+- [Installation](#installation)
+- [Configuration](#configuration)
+- [Development Setup](#development-setup)
```

Comment:
Why do we want to include table of contents everywhere? Isn't it enough that it's already displayed to the right side on the website?

Like here:

![image](https://github.com/janus-idp/backstage-plugins/assets/7453394/d8ec2af2-2c0c-4215-ab7d-8d6063943ec6)

https://janus-idp.io/plugins/ocm

Reply:
I initially noticed that some of the plugme README.md files already had table of contents (ex: tekton, topology) so I thought it would be fine to add it to the other README.md file for the sake of consistency. 

Additionally, I believe the table of contents in the README.md file would help with navigation when reading the README.md on GitHub as it has no built-in ToC.

![image](https://github.com/janus-idp/backstage-plugins/assets/50030060/301ba2b4-99b2-4900-80fb-294c6dbf002b)
https://github.com/janus-idp/backstage-plugins/tree/main/plugins/ocm

Reply:
I don't think we need a ToC because the READMEs are small and it would be redundant on the community website.

Reply:
Ok I will remove them then